### DISCUSSION TITLE
posting to slack channel from github pr comments

### DISCUSSION THREAD

**Discussion by irenerhee** [community/community#33851] [Actions] [answered]
hi, was wondering if anyone knew how to post to a slack channel from a github pr comment? i know this is possible because ive seen it before.

ex flow: 
- have slack channel reserved for pr's, e.g. `#eng-pr`
- i open new pr
- i want to request reviews on it, so make comment prefaced with special emoji, e.g. `👀 ptal`
- a slack msg appears in `#eng-pr` saying `irene posted in <link to git pr>: 👀 ptal`

i tried looking for this functionality online / instructions on how to set this up but i dont see any

**Comment by DenverCoder1** [comment, marked as answer]
There are many ways including bots, zapier Integration, forwarding webhooks, or using GitHub actions to send to Slack webhooks.

I haven't done it myself, but it looks like the best way would be to create an Incoming Webhook for your Slack channel and use a GitHub Action such as [slack-incoming-webhook](https://github.com/marketplace/actions/slack-incoming-webhook) to send to the Webhook on a new PR.

You'd want to add the Webhook in your repository secrets, and you should be able to find examples of workflows that run on pull request.
